can_tools/scrapers/official/TN/tn_vaccine.py

```diff
@@ -1,44 +1,43 @@
 import pandas as pd
-import requests
```

Comment:
[Tennessee's dashboard ](https://www.tn.gov/health/cedep/ncov/covid-19-vaccine.html) seems to have gone offline, but they have a CSV source that's probably better to pull from anyway, so I've updated this to use it instead